src/components/TokenSelector.tsx

```diff
@@ -52,7 +59,16 @@ const TokenSelector: React.FC = () => {
     <StyledWrapper>
       <StyledInput type="number" placeholder="0" />
       <StyledTokenSelect onClick={handleOpenModal}>
-        Select Token
+        <div>Select Token</div>
+        <svg
```

Comment:
please put svg in separate file. Then you can import like thsi

import {ReactComponent as MySvgIcon} from './your-icon.svg';